packages/main/src/components/save-password/SavePassword.jsx

```diff
@@ -1,27 +1,87 @@
-import React, { useRef } from "react";
-import useForm from "../use-form-hook";
+import React, { useRef, useState } from "react";
 import styles from "../save-password/SettingsTab.module.css";
 import { BASE_API_URL } from "@zuri/utilities";
 
+let errorOut = null;
+
 const SavePassword = () => {
   const formElement = useRef(null);
+  const [state, setState] = useState({ status: "idle", message: null });
   const user = JSON.parse(sessionStorage.getItem("user"));
   const FORM_ENDPOINT = `${BASE_API_URL}/users/${user.id}`;
-  const { message, handleSubmit } = useForm({
-    form: formElement.current,
-    options: {
-      method: "PATCH",
-      credentials: "include",
+
+  const checkPassword = async password => {
+    const confirmPassword = {
+      password,
+      confirm_password: password
+    };
+    const response = await fetch(`${BASE_API_URL}/auth/confirm-password`, {
+      method: "POST",
       headers: {
         Accept: "application/json",
         Authorization: `Bearer ${user.token}`,
         "Content-Type": "application/json"
-      }
-    },
-    endpointUrl: FORM_ENDPOINT
-  });
+      },
+      body: JSON.stringify(confirmPassword)
+    });
+
+    const data = await response.json();
+
+    if (!response.ok) {
+      throw new Error(data.message);
```

Comment:
Use toat notifications to show errors.

Reply:
I am now using toast please review again @billmal071